edu.cuny.hunter.streamrefactoring.core/META-INF/MANIFEST.MF

```diff
@@ -22,24 +18,18 @@ Require-Bundle: org.eclipse.ltk.core.refactoring,
  com.ibm.wala.shrike;bundle-version="1.3.10",
  com.ibm.safe.core;bundle-version="1.0.0",
  com.ibm.safe.typestate;bundle-version="1.0.0",
- org.eclipse.jdt.launching
+ org.eclipse.jdt.launching,
+ org.eclipse.core.resources,
+ org.eclipse.core.runtime,
+ org.eclipse.jdt.core.manipulation,
+ org.eclipse.text,
+ org.eclipse.ui,
+ org.eclipse.pde.core,
+ org.objenesis
```

Comment:
How does this get here? I see that the jar is removed, but now it is being referenced as a plug-in. Where did the plug-in come from?

Reply:
I guess you mean the objenesis dependency.

The plugin is resolved directly in the Nexus central : 
https://github.com/yanntm/Optimize-Java-8-Streams-Refactoring/blob/master/edu.cuny.hunter.streamrefactoring.parent/pom.xml#L149-L152

A lot of third-party artifacts are released there, among which is this objenesis dependency. https://mvnrepository.com/artifact/org.objenesis/objenesis 

This is not as good as grabbing it from Orbit, because Orbit is a "simultaneous release" of components that are tested and interact well together and with the version of eclipse platform we are targetting.  But it avoids at least repackaging our own, and lets us update automatically if a new version is released to maven central.  It certainly is better than having a copy of the jar in our lib/ folder.